src/travel-planner/model/model.go

```diff
@@ -34,8 +34,58 @@ type Vacation struct {
 }
 
 type User struct {
-	gorm.Model
-	Id       uint
-	Email    string
-	Password string
+
+	Id       uint32    `json:"id"`
+	Email    string  `json:"email"`
+	Password string   `json:"password"`
+	Username string `json:"username"`
+	Age      int64  `json:"age"`
+	Gender   string `json:"gender"`
```

Comment:
Using 0/1 indicator - 0 for Male, 1 for Female. Change `Gender string` to `Gender int`